### PR TITLE
Roll CI log

### PR DESCRIPTION
**Pull request by AlCutter**
This PR updates GCB configs for the newly rolled CI log.

See also:
- https://github.com/transparency-dev/armored-witness/pull/67
- https://github.com/transparency-dev/armored-witness-applet/pull/185

### CODE REVIEW

**Review by mhutchinson — APPROVED** [review]

File: release/cloudbuild_withsecure_signature.yaml [hunk 1]
```diff
@@ -52,7 +52,7 @@ steps:
       - call
       - sequence
       - --data
-      - '{"entriesDir": "${_ENTRIES_DIR}", "origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": 1, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
+      - '{"entriesDir": "${_ENTRIES_DIR}", "origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": ${KEY_VERSION}, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
```
Comment (jiggoha) [inline release/cloudbuild_withsecure_signature.yaml:55]: I think this is missing an underscore here and below.

File: release/cloudbuild_withsecure_signature.yaml [hunk 2]
```diff
@@ -61,17 +61,19 @@ steps:
...
   # Log-related.
   _ENTRIES_DIR: firmware-log-sequence
   # TODO(jayhou): switch to prod origin.
-  _ORIGIN: transparency.dev/armored-witness/firmware_transparency/ci/0
+  _ORIGIN: transparency.dev/armored-witness/firmware_transparency/ci/1
   # TODO(jayhou): do not use CI bucket when we flip this trigger to prod.
-  _LOG_NAME: armored-witness-firmware-log-ci
\ No newline at end of file
+  _LOG_NAME: armored-witness-firmware-log-ci-1
+  # This must correspond with the trailing number on the _FIRMWARE_BUCKET, _ORIGIN, _LOG_NAME values.
```
Comment (jiggoha) [inline release/cloudbuild_withsecure_signature.yaml:78]: Would it be worth breaking up _LOG_NAME and _ORIGIN, and _FIRMWARE_BUCKET to drop the `-1` and just replace them with `{_LOG_NAME}-{_KEY_VERSION}`?

File: release/cloudbuild_withsecure_signature.yaml [hunk 3]
```diff
@@ -61,17 +61,19 @@ steps:
       - call
       - integrate
       - --data
-      - '{"origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": 1, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
+      - '{"origin": "${_ORIGIN}", "bucket": "${_LOG_NAME}", "kmsKeyName": "ft-log-ci", "kmsKeyRing": "firmware-release-ci", "kmsKeyVersion": ${KEY_VERSION}, "kmsKeyLocation": "global", "noteKeyName": "transparency.dev-aw-ftlog-ci"}'
```
Comment (jiggoha) [inline release/cloudbuild_withsecure_signature.yaml:64]: Does the noteKeyName also need to include the log version?

File: release/cloudbuild_withsecure_signature.yaml (hunk 2 above)
Comment (AlCutter) [reply release/cloudbuild_withsecure_signature.yaml:78]: Does that work inside the `substitutions` section?

File: release/cloudbuild_withsecure_signature.yaml (hunk 1 above)
Comment (AlCutter) [reply release/cloudbuild_withsecure_signature.yaml:55]: Good spot, thanks!

File: release/cloudbuild_withsecure_signature.yaml (hunk 2 above)
Comment (AlCutter) [reply release/cloudbuild_withsecure_signature.yaml:78]: Ah, you mean where these subs are used...

**Pull request merged**